app/medInria/main.cpp

```diff
@@ -125,9 +107,9 @@ int main(int argc,char* argv[]) {
     dtkLogger::instance().attachFile(dtkLogPath(&application));
     dtkLogger::instance().attachConsole();
 
-    qDebug() << "####################################";
-    qDebug() << "Version: "    << MEDINRIA_VERSION;
-    qDebug() << "Build Date: " << MEDINRIA_BUILD_DATE;
+    dtkDebug() << "####################################";
+    dtkDebug() << "Version: "    << MEDINRIA_VERSION;
+    dtkDebug() << "Build Date: " << MEDINRIA_BUILD_DATE;
 
```

Comment:
In order to fully understand, you changed here `qDebug` to `dtkDebug`. We don't need to change it everywhere in the code, right?